frontend/src/app/modules/dashboard/views/chat/components/chat-contacts/components/chat-contact-item/chat-contact-item.component.scss

```diff
@@ -14,7 +16,7 @@
   position: relative;
   display: flex;
 
-  background-color: #eff2f4;
+  background-color: $chat-bg-color;
```

Comment:
Daca sa zicem ca e o exceptie de culoare, gen ai pus-o specific $chat-bg-color atunci pe nimeni nu mai intereseaza si ar trebui sa fie sus aici sub importuri variabila declarata , nu in _colors.scss

Reply:
fixed